clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -965,6 +1100,70 @@ CIRGenFunction::buildPointerToDataMemberBinaryExpr(const BinaryOperator *E) {
   return makeAddrLValue(memberAddr, memberPtrTy->getPointeeType(), baseInfo);
 }
 
+LValue
+CIRGenFunction::buildExtVectorElementExpr(const ExtVectorElementExpr *E) {
+  // Emit the base vector as an l-value.
+  LValue Base;
```

Comment:
Same for this function, please make sure you add testcases or `llvm_unreachable("NYI");` for paths that are not tested.